src/Misc/layoutroot/env.sh

```diff
@@ -12,6 +12,8 @@ varCheckList=(
     'VSTS_HTTP_PROXY'
     'VSTS_HTTP_PROXY_USERNAME'
     'VSTS_HTTP_PROXY_PASSWORD'
+    'LD_LIBRARY_PATH'
+    'PERL5LIB'
```

Comment:
why we need `PERL5LIB`?

Reply:
Part of git via SCL, it's needed for at least part of git.

Reply:
Tossed together a CentOS7 VM to look real quick, it's used by one git hook and a bunch of example hooks:

```
[root@centosvm ~]# rpm -qR rh-git29-git
/bin/sh
/usr/bin/perl
emacs-filesystem >= 24.3
libc.so.6()(64bit)
libc.so.6(GLIBC_2.14)(64bit)
libc.so.6(GLIBC_2.2.5)(64bit)
libc.so.6(GLIBC_2.3)(64bit)
libc.so.6(GLIBC_2.3.4)(64bit)
libc.so.6(GLIBC_2.4)(64bit)
libc.so.6(GLIBC_2.7)(64bit)
libpcre.so.1()(64bit)
libpthread.so.0()(64bit)
libpthread.so.0(GLIBC_2.2.5)(64bit)
libpthread.so.0(GLIBC_2.3.2)(64bit)
librt.so.1()(64bit)
librt.so.1(GLIBC_2.2.5)(64bit)
libz.so.1()(64bit)
libz.so.1(ZLIB_1.2.0)(64bit)
perl >= 0:5.008
perl(Error)
perl(Error)
perl(File::Basename)
perl(File::Copy)
perl(File::Find)
perl(File::Path)
perl(File::Spec)
perl(File::Temp)
perl(File::stat)
perl(Getopt::Long)
perl(Term::ReadKey)
perl(lib)
perl(strict)
perl(warnings)
rh-git29-git-core = 2.9.3-8.el7
rh-git29-git-core-doc = 2.9.3-8.el7
rh-git29-perl(Git)                                <---- HERE
rh-git29-perl-Git = 2.9.3-8.el7                   <---- HERE
rh-git29-runtime
rh-git29-runtime
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PartialHardlinkSets) <= 4.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rtld(GNU_HASH)
rpmlib(PayloadIsXz) <= 5.2-1
```